force-app/main/default/lwc/geFormField/geFormField.js

```diff
@@ -376,8 +376,20 @@ export default class GeFormField extends LightningElement {
     }
 
     set formState(formState) {
```

Comment:
Could a jest test be added to test this?

Reply:
Yeah! Pushed up a new commit. Had to extract the block to a helper file so we can spy on it and confirm that it was called in the jest test.